misc/python/materialize/test_analytics/connector/test_analytics_connector.py

```diff
@@ -101,6 +100,7 @@ def create_cursor(
                 connection = self.create_connection(autocommit=autocommit)
 
         cursor = connection.cursor()
+        cursor.autocommit = True
```

Comment:
```suggestion
        cursor.autocommit = autocommit
```

Reply:
(Or we remove the param altogether. I wanted to use explicit txs but learnt that this will prevent me from doing `INSERT INTO ... SELECT`.)